ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.18.0__46.yaml

```diff
@@ -127,6 +127,11 @@ tests:
   commands: make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: e2e-aws-ocp-45-continuous
```

Comment:
Should this be  e2e-aws-ocp-46-continuous ?

Reply:
:stuck_out_tongue: